deps/fdasrsf/optimum_reparam.pyx

```diff
@@ -1,4 +1,5 @@
-# cython: language_level=3
+#!python
```

Comment:
Why the shebang? And why Python?

Reply:
Looking for solutions to #78 I saw a cython code with the same problem and the code fixed had this shebang.
The problem was because the extension and the .pyx file had a different name, doesn't change anything, but I preferred not to change it just in case.